openmls/src/group/public_group/staged_commit.rs

```diff
@@ -280,10 +280,12 @@ impl PublicGroup {
     ///
     /// This function should not fail and only returns a [`Result`], because it
     /// might throw a `LibraryError`.
+    /// TODO: fix comment above. Also, should we write to storage here? in that
```

Comment:
The TODO assumed that we return an error, which we didn't, because we didn't actually write to storage. Now fixed that we write to storage, and I just got rid of the section of the comment that described what kind of error we return.